aws/data.yaml

```diff
@@ -2,30 +2,32 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: data
+  name: data-dev
   labels:
     environment: dev
     app: data
 spec:
   selector:
     matchLabels:
+      environment: dev
       app: data
   replicas: 1
   template:
     metadata:
       labels:
+        environment: dev
         app: data
     spec:
       containers:
         - name: data
-          image: docker.io/healthmapidha/dataservice:latest
+          image: docker.io/healthmapidha/dataservice:0.0.3
```

Comment:
Shouldn't this be some sort of tag, rather than a concrete version, so that the image can be updated without committing changes to this config file? I haven't researched this -- just asking based on notions.

Reply:
constant tags are not considered best practices because new images will be pulled only when a container is restarted, not when a new tag is applied to the image (kubernetes master does not watch docker hub). I sent you a chat with a link to why this is also considered bad practice internally.

Reply:
Makes sense for prod, but I'm asking specifically re: the change from latest to 0.0.3 for dev. Do we want to update/push this file prior to performing a restart every time we want to update dev? Perhaps there's a way to automate this?

Reply:
For dev we can keep using `:latest` but I am not sure how to trigger an application of that deployment upon the build of a new image, I'll look into docker hub hooks.